doctor_appointment_frontend/src/components/Register.js

```diff
@@ -44,9 +44,7 @@ function Register() {
       .then((data) => {
         // eslint-disable-next-line no-console
         console.log('Success:', data);
-        // eslint-disable-next-line no-alert
-        alert('Appointment booked successfully');
-        navigate('/');
+        navigate('/sing_up');
```

Comment:
It would be better after the user sign up. The user is logged in and routed to the main page "doctors list"